packages/docs/docs/core-api/data-schema.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Data Schema 
+---
+
+User-defined data schemas can be used when interfacing with the exposed `uint253` attestation data. This tool encodes and decodes attestation data, consolidating a user-defined attestation changes. 
```

Comment:
This entry section needs some example code showing importing the module, an example schema, and using the functions containing in here.

Reply:
Example added